src/index.ts

```diff
@@ -1,2 +1,3 @@
 export { GenericContainer } from "./generic-container";
+export { TestContainer, StartedTestContainer, StoppedTestContainer } from "./test-container";
```

Comment:
If you also export DEFAULT_STOP_OPTIONS then users can just override the parts they care about, eg
```ts
await testContainer.stop({
  ...DEFAULT_STOP_OPTIONS,
  timeout: 0
});
```

Reply:
Alternatively, you could do a merge between the passed options and the full set of options, which is probably nicer.